src/api/package-api.ts

```diff
@@ -26,6 +27,46 @@ class PackageApi {
         });
     }
 
+    public async findActiveVersionById(nodeId: string): Promise<PackageHistoryTransport> {
+        return httpClientV2.get(`/package-manager/api/packages/${nodeId}/active`).catch(e => {
+            throw new FatalError(`Problem getting latest version of package: ${e}`);
+        });
+    }
+
+    public async findNextVersion(nodeId: string): Promise<PackageHistoryTransport> {
+        return httpClientV2.get(`/package-manager/api/packages/${nodeId}/next-version`).catch(e => {
+            throw new FatalError(`Problem getting latest version of package: ${e}`);
+        });
+    }
+
+    public async findOneByKeyAndRootNodeKey(packageKey: string): Promise<ContentNodeTransport> {
+        return httpClientV2.get(`/package-manager/api/nodes/${packageKey}/${packageKey}`).catch(e => {
+            return null
+        });
+    }
+
+    public async importPackage(nodeContent: any, nodeId: string, spaceId: string, overwrite: boolean): Promise<any> {
+        await httpClientV2.post(`/package-manager/api/packages/import?spaceId=${spaceId}` + (overwrite ? "&overwrite=true" : ""), nodeContent).catch(e => {
```

Comment:
Instead of building URI strings, can we pass in the request params to the httpClient and let it handle building the URIs.